scripts/classify-failures

```diff
@@ -0,0 +1,254 @@
+#!/usr/bin/python3
+#
+# Copyright (C) 2023  Red Hat, Inc.
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# the GNU General Public License v.2, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY expressed or implied, including the implied warranties of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.  You should have received a copy of the
+# GNU General Public License along with this program; if not, write to the
+# Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301, USA.  Any Red Hat trademarks that are incorporated in the
+# source code or documentation are not subject to the GNU General Public
+# License and may only be used or replicated with the express permission of
+# Red Hat, Inc.
+
+import argparse
+import subprocess
+import sys
+
+ISSUES = [
+    (
+        "<ISSUE DESCRIPTION PREFERABLY WITH GITHUB ISSUE REFERENCE>",
+        "<REGULAR EXPRESSION FOR GREP TO BE LOOKED FOR>",
+    ),
+    (
+        # The detection may be improved, not sure if just with grep.
+        "[1] https://github.com/rhinstaller/kickstart-tests/issues/675 The string is also in successful installation, but there is high chance that for test which times out it will be this issue.",
```

Comment:
I'm not sure I understand this - is it `[1]` or `[675]` ?

Reply:
I'll change it to [675] (the [1] is obsolete tracking)